main.js

```diff
@@ -0,0 +1,5 @@
+document.querySelector("li").remove();
+document.querySelector("span").remove();
+document.querySelector("main .posts article").remove();
```

Comment:
Gotta remove the last article, not the first one

Reply:
Thank you. I have updated my code.